src/safeds_stubgen/api_analyzer/_type_source_preference.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from enum import Enum
+
+
+class TypeSourcePreference(Enum):
+    CODE = "code"
+    DOCSTRING = "docstring"
+    THROW_WARNING = "throw_warning"
```

Comment:
Which type do you use in case of a conflict, if this option is selected?

Seems to be code according to the later files. Because of this, I would strongly suggest splitting this option into two:

1. Preference code vs. docstring.
2. Whether to warn in case of conflicts.

Currently, the "warn" option is more like "code_but_warn".

Reply:
I split this option in two.